website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -52,8 +52,8 @@ Once the connection is saved, a public key will be generated and displayed for t
 The following fields are required when creating a Snowflake connection:
 
 | Field | Description | Examples |
-| ----- | ----------- | -------- |
-| Account | The Snowflake account to connect to. Take a look [here](snowflake-profile#account) to determine what the account field should look like based on your region.| `db5261993`,`db5261993.east-us-2.azure` |
+| ----- | ----------- | ---------------- |
```

Comment:
@mirnawong1 this is workign now! I know there's a way to make the last column wider in markdown, but I can't remember how. If you want to add that, then maybe you can look into it? Otherwise this looks great!